### PR TITLE
getMinutes reduced function

### PR DESCRIPTION
**Pull request by Angelk90**
@aadilhasan : 

On the same genre hour you could write like this:

```
const hours = ["12"].concat(
  [...Array(11).keys()].map(_ => (_ + 1).toString().padStart(2, "0"))
);
```

### CODE REVIEW

File: src/components/time-picker/index.js [hunk 1]
```diff
@@ -2,16 +2,8 @@ import React from 'react';
...
-  let i = 0,
-    limit = 60,
-    minutes = [];
-  for (i; i < limit; i += 1) {
-    minutes.push(i < 10 ? '0' + i : '' + i);
-  }
-  return minutes;
-};
 
+const minutes = [...Array(60).keys()].map(_ => _.toString().padStart(2, "0"));
```
Comment (aadilhasan) [inline src/components/time-picker/index.js]: can you please make **_** to some appropriate variable name like **minute**. proper naming makes it more understandable, thought this code is pretty small and self explanatory, still following the same convention everywhere keeps things uniform.

**Comment by Angelk90** [comment]
@aadilhasan :

I'm not sure I understood correctly.

Do you want me to change the name of:
`const minutes = [...Array(60).keys()].map(_ => _.toString().padStart(2, "0"));
`
in
`const minute = [...Array(60).keys()].map(_ => _.toString().padStart(2, "0"));
`

**Comment by aadilhasan** [comment]
@Angelk90  what i mean is in the map's callback, parameter variable name should have some meaning instead of making it **_** make it **minute**. which is more readable.

**Comment by Angelk90** [comment]
@aadilhasan : Okay, see if it's okay.

**Pull request closed without merging**